rakelib/update_cloud_images.rake

```diff
@@ -65,8 +65,9 @@ def demo_amis(version)
 end
 
 def get_amis(type, version)
-  amis    = []
-  regions = Aws.partition('aws').regions.map {|region| region.name}.delete_if {|region| region == 'ap-east-1'}
+  amis = []
+  # regions = Aws.partition('aws').regions.map {|region| region.name}.delete_if {|region| region == 'ap-east-1'}
+  regions = [env("REGION"), env("EXTRA_AMI_REGION_TO_COPY_TO")]
```

Comment:
I guess this makes sense. We don't show any other regions in https://www.gocd.org/download/#ami

I'd remove the comment.

Reply:
Done